### ISSUE TITLE
calling pylibdmtx.decode on certain images kills the Python process

### ISSUE THREAD

**Issue by jbweston** [NaturalHistoryMuseum/pylibdmtx#25]
## Description
I have noticed that when attempting to decode a particular image (which does not contain a Datamatrix) that calling `pylibdmtx.decode` will kill the Python process with exit code 40.

## Steps to reproduce

Run the following in a Python interpreter:
```python
from PIL import Image
from pylibdmtx import pylibdmtx
pylibdmtx.decode(Image.open('bad_image.png'))
```
with the following image saved as `bad_image.png` in the current working directory;
![bad_image](https://user-images.githubusercontent.com/6654709/43106381-f7ea099c-8ec7-11e8-8730-4ce4f2c799df.png)

### System info

**OS**: Debian GNU/Linux testing (buster) x86_64
**libdmtx version**: `0.7.4` (installed from the Debian repositories and reported by `apt info` as `0.7.4-2+b1`)
**Python version**: `3.6.0`
**pylibdmtx version**:  `0.1.7`
**PIL version**: `5.0.0`

## Expected behavior
An error  should be raised, as the image does not contain a datamatrix

## Observed Behavior
Python process exits with code 40

## Additional information
While this is almost certainly a bug in the underlying `libdmtx` (I have already [opened a bug report](https://github.com/dmtx/libdmtx/issues/16) there), it is perhaps possible that this error could be caught before it hits `libdmtx`, which would allow us to work around the issue for the time being. I have not looked in to the root cause sufficiently deeply to be able to tell if this is possible.

**Comment by ondraga** [comment]
Hi

I need to decode the numbers from a Datamatrix. 
This is my code

import cv2
from pylibdmtx.pylibdmtx import decode
a = decode(cv2.imread(suganth.jpg'))

when trying to print using print(a), it gives me an empty matrix. 
I couldn't understand the decode function. 

Can you help me in decoding any Datamatrix in numbers?

**Comment by jbweston** [comment]
@ondraga this is not the correct place to ask general questions about using `pylibdmtx`. Please remove your comment and ask through a more appropriate channel.

**Comment by jbweston** [comment]
This has been fixed upstream by https://github.com/dmtx/libdmtx/pull/17, which will be included into the upstream v0.7.6 release.

**Comment by jbweston** [comment]
So I'll close the issue here.

**Comment by darouwan** [comment]
> This has been fixed upstream by [dmtx/libdmtx#17](https://github.com/dmtx/libdmtx/pull/17), which will be included into the upstream v0.7.6 release.

0.7.6 hasn't been released yet? Or before the next release, how can we handle this issue?

**Comment by msva** [comment]
Sorry for long reaction.

I just pushed 0.7.7 tag, so it can be taken by downstream maintainers to build packaged versions.
